components/DatePicker/DatePicker.js

```diff
@@ -0,0 +1,62 @@
+import React, {useState} from 'react';
```

Comment:
Looking forward to digging into this component and seeing if I can get this to work for the 'filter by date' functionality as well